### PR TITLE
[New Rule] Attempt to Remove File Quarantine Attribute

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/673

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/defense_evasion_attempt_del_quarantine_attrib.toml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+creation_date = "2020/08/14"
+maturity = "production"
+updated_date = "2020/11/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+In macOS, when applications or programs are downloaded from the internet, there is a special attribute set on the file.
+This attribute is read by Apple's Gatekeeper defense program at execution time. This may indicate a Potential Gatekeeper
+Bypass.
```
Comment (bm11100) [inline rules/macos/defense_evasion_attempt_del_quarantine_attrib.toml:11]:
```suggestion
Identifies a potential Gatekeeper
bypass. In macOS, when applications or programs are downloaded from the internet, there is a quarantine flag set on the file.
This attribute is read by Apple's Gatekeeper defense program at execution time. An adversary may disable this attribute to evade defenses.
```

Comment (bm11100) [reply rules/macos/defense_evasion_attempt_del_quarantine_attrib.toml:11]: will need to re-lint

**Review by bm11100 — APPROVED** [review]

File: rules/macos/defense_evasion_attempt_del_quarantine_attrib.toml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+]
+risk_score = 43
+rule_id = "f0b48bbc-549e-4bcf-8ee0-a7a72586c6a7"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+  process.name:"xattr" and process.args:"com.apple.quarantine" and process.args:"-d"
```
Comment (brokensound77) [inline rules/macos/defense_evasion_attempt_del_quarantine_attrib.toml:29]:
```suggestion
  process.name == "xattr" and process.args == "com.apple.quarantine" and process.args == "-d"
```

**Review by brokensound77 — APPROVED** [review]
Comment on the query then LGTM

**Pull request merged**